library/src/morpheus-master/sass/modules/tool/_base.scss

```diff
@@ -174,8 +174,10 @@
 	}
 
   sakai-maximise-button {
```

Comment:
I can't tell if it's the way I'm doing from end dev via chrome local overrides or not. Do these styles actually make it through to the shadow dom?

Reply:
No, they don't. markup in shadow dom can only be styled with the exposed styles and you have to code those explicitly when writing your webcomponents. However, most of our current components use light dom and pick up all the css you write, just like any other Sakai markup.

Reply:
You've got a sass compilation error going on, breaking the travis build.

io.bit3.jsass.CompilationException: Error: no mixin named sakai-link-button
        on line 133 of library/src/morpheus-master/sass/modules/tool/_base.scss
        from line 16 of library/src/morpheus-master/sass/tool.scss
>> 		@include sakai_link_button();

Reply:
forgot to add the extendables change. I'll update and recommit.